tests/test.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pycurl
```

Comment:
hmm? why not requests? http://docs.python-requests.org/en/master/

Reply:
I copied the test from the model and edited it. It would probably be more consistent to use requirements though wouldn't it, I'll see what it would take to switch.

Reply:
okay, i didn't realize it came from the model. whatever is easier for you.

Reply:
Perhaps the model tests could look at using `requests` instead? cc @bdwyer2

Reply:
I'm currently rewriting it with requests if you want to take my updates once I'm done and use them for the models

Reply:
`requests` is the most "standard" http library for python as far as I've seen, it seems to me that `pycurl`s purpose is the replicate the curl command inside python rather than be a http library.

Reply:
i'll admit i'm being biased :)

requests will handle a lot of the ugly stuff and in general make things cleaner/shorter: https://github.com/yudazilian/Pycurl-vs-Requests

requests also has a nice library for mocking requests: https://github.com/jamielennox/requests-mock should you ever want to mock things instead of testing against a live server.